src/test/resources/xmlEmptyElementsNoNull.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <root>
-    <boolElement>true</boolElement>
     <emptyElement/>
+    <valueElement>Test</valueElement>
```

Comment:
What's with these changes and the ones in the other XML file?

Reply:
Ah, these are expected results, since the order is preserved now, the test results didn't match the expected ones, so I updated it.